waku/common/databases/db_postgres/dbconn.nim

```diff
@@ -235,6 +235,49 @@ proc isSecureString(input: string): bool =
 
   return true
 
+proc convertQueryToMetricLabel(query: string): string =
+  ## Simple query categorization. The output label is the one that should be used in query metrics
+  if query.contains("contentTopic IN"):
+    return ContentTopicQuery
+  elif query.contains("SELECT version()"):
+    return SelectVersionQuery
+  elif query.contains("WITH min_timestamp"):
+    return MessagesLookupQuery
```

Comment:
Don't you have like a key-value map in Nim?

Reply:
Yeah, but can't you just use keys as prefixes and iterate over them?

Reply:
what @jakubgs says.